### ISSUE TITLE
[Bug]: NLTk not working

### ISSUE THREAD

**Issue by tap0212** [run-llama/llama_index#15566]
### Bug Description

NLTK punnk not wotking

### Version

0.10.68

### Steps to Reproduce

When running

### Relevant Logs/Tracbacks

```shell
app-1          | [nltk_data] Downloading package punkt_tab to /py/lib/python3.9/site-
app-1          | [nltk_data]     packages/llama_index/core/_static/nltk_cache...
app-1          | Traceback (most recent call last):
app-1          |   File "/py/lib/python3.9/site-packages/llama_index/core/utils.py", line 65, in __init__
app-1          |     nltk.data.find("tokenizers/punkt")
app-1          |   File "/py/lib/python3.9/site-packages/nltk/data.py", line 579, in find
app-1          |     raise LookupError(resource_not_found)
app-1          | LookupError:
app-1          | **********************************************************************
app-1          |   Resource punkt not found.
app-1          |   Please use the NLTK Downloader to obtain the resource:
app-1          |
app-1          |   >>> import nltk
app-1          |   >>> nltk.download('punkt')
app-1          |
app-1          |   For more information see: https://www.nltk.org/data.html
app-1          |
app-1          |   Attempted to load tokenizers/punkt
app-1          |
app-1          |   Searched in:
app-1          |     - '/home/django-user/nltk_data'
app-1          |     - '/py/nltk_data'
app-1          |     - '/py/share/nltk_data'
app-1          |     - '/py/lib/nltk_data'
app-1          |     - '/usr/share/nltk_data'
app-1          |     - '/usr/local/share/nltk_data'
app-1          |     - '/usr/lib/nltk_data'
app-1          |     - '/usr/local/lib/nltk_data'
app-1          |     - '/py/lib/python3.9/site-packages/llama_index/core/_static/nltk_cache'
app-1          | **********************************************************************
app-1          |
app-1          |
app-1          | During handling of the above exception, another exception occurred:
app-1          |
app-1          | Traceback (most recent call last):
app-1          |   File "/app/manage.py", line 22, in <module>
app-1          |     main()
app-1          |   File "/app/manage.py", line 18, in main
app-1          |     execute_from_command_line(sys.argv)
app-1          |   File "/py/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
app-1          |     utility.execute()
app-1          |   File "/py/lib/python3.9/site-packages/django/core/management/__init__.py", line 363, in execute
app-1          |     settings.INSTALLED_APPS
app-1          |   File "/py/lib/python3.9/site-packages/django/conf/__init__.py", line 82, in __getattr__
app-1          |     self._setup(name)
app-1          |   File "/py/lib/python3.9/site-packages/django/conf/__init__.py", line 69, in _setup
app-1          |     self._wrapped = Settings(settings_module)
app-1          |   File "/py/lib/python3.9/site-packages/django/conf/__init__.py", line 170, in __init__
app-1          |     mod = importlib.import_module(self.SETTINGS_MODULE)
app-1          |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
app-1          |     return _bootstrap._gcd_import(name[level:], package, level)
app-1          |   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
app-1          |   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
app-1          |   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
app-1          |   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
app-1          |   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
app-1          |   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
app-1          |   File "/app/app/settings.py", line 19, in <module>
app-1          |     from llama_index.llms.azure_openai import AzureOpenAI
app-1          |   File "/py/lib/python3.9/site-packages/llama_index/llms/azure_openai/__init__.py", line 1, in <module>
app-1          |     from llama_index.llms.azure_openai.base import (
app-1          |   File "/py/lib/python3.9/site-packages/llama_index/llms/azure_openai/base.py", line 4, in <module>
app-1          |     from llama_index.core.base.llms.types import ChatMessage
app-1          |   File "/py/lib/python3.9/site-packages/llama_index/core/__init__.py", line 10, in <module>
app-1          |     from llama_index.core.base.response.schema import Response
app-1          |   File "/py/lib/python3.9/site-packages/llama_index/core/base/response/schema.py", line 9, in <module>
app-1          |     from llama_index.core.schema import NodeWithScore
app-1          |   File "/py/lib/python3.9/site-packages/llama_index/core/schema.py", line 18, in <module>
app-1          |     from llama_index.core.utils import SAMPLE_TEXT, truncate_text
app-1          |   File "/py/lib/python3.9/site-packages/llama_index/core/utils.py", line 89, in <module>
app-1          |     globals_helper = GlobalsHelper()
app-1          |   File "/py/lib/python3.9/site-packages/llama_index/core/utils.py", line 67, in __init__
app-1          |     nltk.download("punkt_tab", download_dir=self._nltk_data_dir)
app-1          |   File "/py/lib/python3.9/site-packages/nltk/downloader.py", line 774, in download
app-1          |     for msg in self.incr_download(info_or_id, download_dir, force):
app-1          |   File "/py/lib/python3.9/site-packages/nltk/downloader.py", line 642, in incr_download
app-1          |     yield from self._download_package(info, download_dir, force)
app-1          |   File "/py/lib/python3.9/site-packages/nltk/downloader.py", line 699, in _download_package
app-1          |     os.makedirs(os.path.join(download_dir, info.subdir), exist_ok=True)
app-1          |   File "/usr/local/lib/python3.9/os.py", line 225, in makedirs
app-1          |     mkdir(name, mode)
app-1          | PermissionError: [Errno 13] Permission denied: '/py/lib/python3.9/site-packages/llama_index/core/_static/nltk_cache/tokenizers'
app-1 exited with code 1
```

**Comment by logan-markewich** [comment]
Notice that this line of code does not match the code in your traceback. The latest version is already fixed 

https://github.com/run-llama/llama_index/blob/ef9a21c73d35f01332cdbfece5a31f4125e2a939/llama-index-core/llama_index/core/utils.py#L65

**Comment by logan-markewich** [comment]
Seems like for whatever reason `pip install llama-index` isn't catching the `llama-index-core==0.10.68.post1` version

This will be fixed shortly today anyways, as v0.11.0 is coming out

**Comment by logan-markewich** [comment]
This has since been fixed in latest versions
